x-pack/plugins/siem/public/components/page/network/ip_overview/field_renderers.tsx

```diff
@@ -79,9 +79,13 @@ export const hostIdRenderer = (host: HostEcsFields, ipFilter?: string): React.Re
   host.id && host.ip && (!(ipFilter != null) || host.ip.includes(ipFilter)) ? (
     <EuiFlexGroup alignItems="center" gutterSize="none">
       <EuiFlexItem grow={false}>
-        <DefaultDraggable id={`${IpOverviewId}-host-id`} field={'host.name'} value={host.name}>
-          <HostDetailsLink hostName={host.name!} />
-        </DefaultDraggable>
+        {host.name != null ? (
+          <DefaultDraggable id={`${IpOverviewId}-host-id`} field={'host.name'} value={host.name}>
+            <HostDetailsLink hostName={host.name}>{host.id}</HostDetailsLink>
+          </DefaultDraggable>
+        ) : (
+          <>host.id</>
+        )}
```

Comment:
Wow! 💯, Like how you are checking to see if `host.name` is `null` and then doing this extra work.

However, did you update the tests and test some this condition?

Reason I ask is it looks like you are just going to show the text of `host.id` rather than `<>{host.id}<>` ;-)

Also, look above at line 79 here:

```
 host.id && host.ip && (
```

Was there a mistake maybe where we wanted to check against `host.name` and not `host.id`?